src/tests/system/tests/test_memory_cache.py

```diff
@@ -1656,3 +1659,366 @@ def test_memory_cache__truncate__nosigbus(client: Client, ldap: LDAP):
     result = client.tools.id("user-1")
     assert result is not None, "User was not found"
     assert result.user.name == "user-1"
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_memory_cache__verify_timestamp_cache(client: Client, provider: GenericProvider):
+    """
+    :title: Verify the existence of timestamp cache and use lsbsearch on those files
+    :setup:
+        1. Add user
+        2. Start SSSD
+    :steps:
+        1. Execute getent passwd to fetch user details
+        2. Check if timestamps cache file exists
+        3. Get user information using ldbsearch on cache_test.ldb
+        4. Get user timestamp information using ldbsearch on timestamps_test.ldb
+    :expectedresults:
+        1. User details should be successfully fetched
+        2. Cache file should be present
+        3. User information were successfully fetched
+        4. User information were successfully fetched
+    :customerscenario: False
+    """
+    provider.user("user1").add()
+    client.sssd.start()
+
+    client.tools.getent.passwd("user1")
+    cache = "/var/lib/sss/db/cache_test.ldb"
+    timestamps = "/var/lib/sss/db/timestamps_test.ldb"
+    assert client.fs.exists(timestamps), f"Timestamp file '{timestamps}' does not exist"
+
+    ldb1 = client.ldb.search(cache, "name=user1@test,cn=users,cn=test,cn=sysdb")
+    ldb2 = client.ldb.search(timestamps, "name=user1@test,cn=users,cn=test,cn=sysdb")
+    assert ldb1 != {}, f"ldbsearch failed to find user1 in {cache}"
+    assert ldb2 != {}, f"ldbsearch failed to find user1 in {timestamps}"
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_memory_cache__expire_cache_verify_updates(client: Client, provider: GenericProvider):
+    """
+    :title: Expire user entries in cache and verify the updates
```

Comment:
Steps below doesn't look like "verify the updates".

Again it rather verifies specific implementation.

Proper testing would check that:
 - 2nd 'getent' before cache expiration doesn't trigger network search
 -  after cache expiration 'getent' for the same user triggers network search
Something like that.

Reply:
We can remove this test entirely.

https://github.com/SSSD/sssd/blob/master/src/tests/system/tests/test_sss_cache.py

Covers the test scenario.